platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/CompositeProjectInitDescriptor.java

```diff
@@ -120,6 +121,8 @@ public Map<String, List<String>> generateWithExternalComments(InitSettings setti
                 generator.generate(settings, buildContentGenerationContext);
             }
         }
-        return ((LanguageSpecificAdaptor) descriptor).generateWithExternalComments(settings, buildContentGenerationContext);
+        Map<String, List<String>> comments = ((LanguageSpecificAdaptor) descriptor).generateWithExternalComments(settings, buildContentGenerationContext);
+        VersionCatalogGenerator.create(settings.getTarget()).generate(buildContentGenerationContext);
+        return comments;
```

Comment:
This was required in order to update the wrapper because https://github.com/gradle/gradle/pull/26587 broke samples generation.

See failures in https://builds.gradle.org/buildConfiguration/Gradle_Release_Check_Stage_ReadyforNightly_Trigger/73883037